docker-compose.yml

```diff
@@ -3,14 +3,15 @@ version: "3.8"
 services:
   booking-service-db:
     container_name: booking-service-db
-    image: mysql
+    image: postgres
     restart: unless-stopped
     env_file: ./.env
     environment:
```

Comment:
Add timezone for our AWS 
![image](https://github.com/lux-fm-team/booking-service/assets/56000560/29379f75-01cf-4031-9a8b-516c9f0d68e4)

Reply:
And in Docker file add 
![image](https://github.com/lux-fm-team/booking-service/assets/56000560/2582c815-d451-430a-96ed-f333bab00e45)